### ISSUE TITLE
Search Error: SQLite3 can only bind numbers...

### ISSUE THREAD

**Issue by dday4thedeceiver** [flightplan-tool/flightplan#10]
This happens after the first search on BA site comes up as not available. Similar result with NH so it's not airline specific.
~~~~
flightplan search 0.2.2
Airline website to search (2-letter code)? BA
Departure city (3-letter code)? STL
Arrival city (3-letter code)? POM
Desired cabin class (first/business/premium/economy)? economy
Start date of search range (YYYY-MM-DD)? 2019-04-01
End date of search range (YYYY-MM-DD)? 2019-07-21
Searching 112 days of award inventory: 2019-04-01 - 2019-07-21
[BA] DEPARTURE [POM -> STL] - 2019-04-01 (1 Passenger)
[BA] info Logging in...
[BA] success Login succeeded
error A fatal error occurred!
TypeError: SQLite3 can only bind numbers, strings, Buffers, and null
    at Object.insertRow (C:\Users\mark\AppData\Local\Yarn\Data\global\node_modules\flightplan-tool\shared\db.js:100:27)
    at Object.saveAwards (C:\Users\mark\AppData\Local\Yarn\Data\global\node_modules\flightplan-tool\shared\utils.js:266:26)
    at main (C:\Users\mark\AppData\Local\Yarn\Data\global\node_modules\flightplan-tool\bin\cli-search.js:318:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
~~~~

**Comment by jd20** [comment]
Do you have an old database lying around from the earlier version of flightplan? Try deleting "db" and "data" folders, and run again.

I ran for just 2019-04-01, and both searching and parsing was successful, so pretty certain this is an incompatibility with the old database format.

```
❯ node bin/cli-search.js -w BA -f STL -t POM -c economy -s 2019-04-01 -e 2019-04-01
Creating database...
Searching 1 days of award inventory: 2019-04-01 - 2019-04-01
[BA] DEPARTURE [POM -> STL] - 2019-04-01 (1 Passenger)
[BA] info Logging in...
[BA] success Login succeeded
[BA] DEPARTURE [STL -> POM] - 2019-04-01 (1 Passenger)
success Search complete!
```

**Comment by dday4thedeceiver** [comment]
Strange. I did delete the db, and subsequently data. I did it again for good measure - as you can see below: 
~~~~
flightplan search 0.2.2
Airline website to search (2-letter code)? BA
Departure city (3-letter code)? STL
Arrival city (3-letter code)? NRT
Desired cabin class (first/business/premium/economy)? economy
Start date of search range (YYYY-MM-DD)? 2019-03-01
End date of search range (YYYY-MM-DD)? 2019-06-30
Creating database...
Searching 121.95833333333333 days of award inventory: 2019-03-01 - 2019-06-30
[BA] DEPARTURE [NRT -> STL] - 2019-03-01 (1 Passenger)
[BA] info Logging in...
[BA] success Login succeeded
error A fatal error occurred!
TypeError: SQLite3 can only bind numbers, strings, Buffers, and null
    at Object.insertRow (C:\Users\mark\AppData\Local\Yarn\Data\global\node_modules\flightplan-tool\shared\db.js:100:27)
    at Object.saveAwards (C:\Users\mark\AppData\Local\Yarn\Data\global\node_modules\flightplan-tool\shared\utils.js:266:26)
    at main (C:\Users\mark\AppData\Local\Yarn\Data\global\node_modules\flightplan-tool\bin\cli-search.js:318:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
~~~~

Did a 1 day search with your exact command line above, same error.

Some issue with SQLite & Windows?

**Comment by jd20** [comment]
Possibly because I run the search / parse commands separately, let me try again.

**Comment by jd20** [comment]
Yep, the error is related to running the search command without the `--no-parser` flag. Debugging it now.

**Comment by jd20** [comment]
Pushed v0.2.3, which should fix this issue. Can you upgrade and try?

**Comment by dday4thedeceiver** [comment]
Yes, it's fixed. Thanks!
